### ISSUE TITLE
Make UI more accessible

### ISSUE THREAD

**Issue by maracuja-juice** [designmodo/Flat-UI#253]
Not all components are accessible for keyboard users (for example slider, switch)
Every component should be accessible for keyboard users.
